packages/mesos/extra/start_mesos.sh

```diff
@@ -21,7 +21,8 @@ if [[ "${distro}" == 'coreos' ]]; then
        echo "Configuring systemd-networkd to ignore docker bridge and DC/OS overlay interfaces..."
        coreos_networkd_config
 
-       if systemctl is-enabled systemd-networkd > /dev/null; then
+       if systemctl is-active systemd-networkd > /dev/null; then
+          echo "Restarting systemd-networkd"
```

Comment:
Shouldn't we also update `dcos.network`? Here is `dcos.network` in 1.12 branch [1]. As of CoreOS 1688.5.3 (or earlier), systemd-networkd has started managing `dummy` interfaces. `spartan` and `minuteman` links were affected by that new behavior [2].

[1] https://github.com/dcos/dcos/blame/3b8b3837cd12bdbc0853e5cd9e0d635b8ba83623/gen/dcos-config.yaml#L98-L104
[2] https://github.com/dcos/dcos/pull/2833